scala/image.bzl

```diff
@@ -57,14 +57,14 @@ def scala_image(
         **kwargs
     )
 
+    tags = kwargs.get("tags", None)
     base = base or DEFAULT_JAVA_BASE
     for index, dep in enumerate(layers):
         this_name = "%s.%d" % (name, index)
-        jar_dep_layer(name = this_name, base = base, dep = dep)
+        jar_dep_layer(name = this_name, base = base, dep = dep, tags = tags)
```

Comment:
Someone already had `tags` put in a variable for the `jar_app_layer` below. I figured it'd be better to re-use that by putting that above both calls. If you want it inlined instead, it would be nice to do the same for the other reference as well. Let me know and I'll fix it.